File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/GamePresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.GameActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface GamePresenter extends BasePresenter{
    void getGame(Integer id);

    void getGames();

    void attachView(GameActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/GamePresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.impl.GameActivity

/**
 * Created by Igor on 12.04.2018.
 */

interface GamePresenter : BasePresenter {
    fun getGame(id: Int?)

    fun getGames()

    fun attachView(activity: GameActivity)
}